legal-api/src/legal_api/resources/business/business.py

```diff
@@ -38,8 +38,15 @@ class BusinessResource(Resource):
 
     @staticmethod
     @cors.crossdomain(origin='*')
+    @jwt.requires_auth
     def get(identifier):
         """Return a JSON object with meta information about the Service."""
+        # check authorization
+        if not authorized(identifier, jwt, action=['view']):
+            return jsonify({'message':
+                            f'You are not authorized to submit a filing for {identifier}.'}), \
```

Comment:
Is the message 'submit a filing' ok for a get endpoint with view authorization?

Reply:
Good point, switched that to:
f'You are not authorized to view business {identifier}.'

Reply:
Good point, switched that to:
f'You are not authorized to view business {identifier}.'